extensions/amp-story/1.0/amp-story.js

```diff
@@ -2665,11 +2682,15 @@ export class AmpStory extends AMP.BaseElement {
 
   /**
    * Fetches from the CDN or localhost the localization strings.
-   * @param {string} languageCode
+   * @param {string[]} candidateLanguageCodes
    * @private
    */
-  fetchLocalizationStrings_(languageCode) {
+  fetchLocalizationStrings_(candidateLanguageCodes) {
     const localizationService = getLocalizationService(this.element);
+    const languageCode = getSupportedLanguageCode(
+      candidateLanguageCodes,
+      SUPPORTED_LANGUAGES
```

Comment:
Is there a reason to pass this as a param? Couldn't this just be in the story service bundle?

Reply:
no reason besides testing. i can remove it and just reference it